ui/widgets/tooltip.js

```diff
@@ -140,6 +140,8 @@ $.widget( "ui.tooltip", {
 		}
 	},
 
+	_setOptionDisabled: $.noop,
```

Comment:
Why is this a no-op? We should just have the method call `._disable()` or `._enable()` based on the value. The call to `._disable()` in `._create()` can also be removed then.